src/openforms/js/components/admin/form_design/registrations/zgw/OptionsVariablesProperties.js

```diff
@@ -0,0 +1,265 @@
+import {produce} from 'immer';
+import PropTypes from 'prop-types';
+import React, {useContext, useState} from 'react';
+import {FormattedMessage, useIntl} from 'react-intl';
+
+import DeleteIcon from 'components/admin/DeleteIcon';
+import {CustomFieldTemplate} from 'components/admin/RJSFWrapper';
+import {FormContext} from 'components/admin/form_design/Context';
+import ActionButton, {SubmitAction} from 'components/admin/forms/ActionButton';
+import ButtonContainer from 'components/admin/forms/ButtonContainer';
+import ComponentSelection from 'components/admin/forms/ComponentSelection';
+import {TextInput} from 'components/admin/forms/Inputs';
+import SubmitRow from 'components/admin/forms/SubmitRow';
+import {ValidationErrorContext} from 'components/admin/forms/ValidationErrors';
+import {FormModal} from 'components/admin/modals';
+import {ChangelistTableWrapper, HeadColumn, TableRow} from 'components/admin/tables';
+
+import {getErrorMarkup, getFieldErrors} from './utils';
+
+const EMPTY_VARIABLE_PROPERTY = {
+  variablesProperties: [{componentKey: '', eigenshap: ''}],
+};
+
+const HeadColumns = () => {
+  const intl = useIntl();
+
+  const componentText = (
+    <FormattedMessage description="Manage component column title" defaultMessage="Variable" />
+  );
+  const componentHelp = intl.formatMessage({
+    description: 'Manage component help text',
+    defaultMessage: 'The value of the selected field will be the process variable value.',
+  });
```

Comment:
Is there a reason to use `FormattedMessage` and `intl.formatMessage` and not only one of them?